puppet/modules/web/manifests/init.pp

```diff
@@ -84,10 +84,24 @@
   create_resources(web::htpasswd, $htpasswds)
 
   # YUM
+  $yum_directory = '/var/www/vhosts/yum/htdocs'
+  $yum_directory_config = [
+    {
+      path    => $yum_directory,
+      options => ['Indexes', 'FollowSymLinks', 'MultiViews'],
+    },
+    {
+      path     => '.+\.(bz2|gz|rpm)$',
+      provider => 'filesmatch',
+      headers  => 'Set Cache-Control "public, max-age=2592000"',
+    },
+  ]
+
   $yum_attrs = {
     servername      => 'yum.theforeman.org',
-    docroot         => '/var/www/vhosts/yum/htdocs',
+    docroot         => $yum_directory,
     docroot_mode    => '2575',
+    directories     => $yum_directory_config,
     custom_fragment => template('web/yum.conf.erb'),
```

Comment:
In this file we use mod_expires. Can we unify this and add `repomd.xml` to `$yum_directory_config`?

Reply:
We'd loose the `IfModule`, but otherwise, yeah

Reply:
After a second look, no, it's not that easy.

Reply:
(don't trust the docs)